test/build.jl

```diff
@@ -1,14 +1,16 @@
 using BinaryProvider
 using CxxWrap
 
+prefix() =  BinaryProvider.Prefix(dirname(dirname(jlcxx_path)))
```

Comment:
Putting `CxxWrap.jlcxx_path` should fix the CI failure, I think

Reply:
Ah yes, agreed! Thanks, good catch! :)